docs/guides/nwaku/build-source.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Build Nwaku from Source
+---
+
+This guide provides detailed steps to build a `nwaku` node from the source to access the latest development version or a specific commit or tag of `nwaku`. If you prefer a more stable version, [download a pre-compiled binary](https://github.com/waku-org/nwaku/tags) instead.
```

Comment:
You can have a more stable version by compiling the latest releaes.

using pre-compiled binary is more convenient.